pandas/core/arrays/arrow/array.py

```diff
@@ -1083,6 +1090,24 @@ def pyarrow_meth(data, skip_nulls, **kwargs):
 
         if name in ["min", "max", "sum"] and pa.types.is_duration(pa_type):
             result = result.cast(pa_type)
+        if name in ["median", "mean"] and pa.types.is_temporal(pa_type):
+            result = result.cast(pa_type)
+        if name in ["std", "sem"] and pa.types.is_temporal(pa_type):
+            result = result.cast(pa.int64())
```

Comment:
Is this int64 cast needed? I thought the result should already be an int type?

Reply:
im pretty sure its float64 at this point